src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -25,6 +26,12 @@ export enum PopoverCloseSource {
   ScrollOut,
 }
 
+export enum PopoverAutofocusTarget {
+  None,
```

Comment:
It feels redundant but I think we should deprecate the `preventAutofocus` instead. It might take a while to achieve that but having both options will allow a gradual refactor.